ompi/mca/io/romio321/romio/adio/ad_gpfs/ad_gpfs_tuning.c

```diff
@@ -37,13 +37,13 @@ long    bglocklessmpio_f_type;
 int     gpfsmpio_bg_nagg_pset;
 int     gpfsmpio_pthreadio;
 int     gpfsmpio_p2pcontig;
-int     gpfsmpio_write_aggmethod;
```

Comment:
It is confusing that some of these symbols have now a different prefix than the rest.